.github/workflows/integration-test.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           python-version: "3.11"
 
       - name: Set up Golang
-        uses: actions/setup-go@41dfa10bad2bb2ae585af6ee5bb4d7d973ad74ed # 5.1.0
+        uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # 5.5.0
```

Comment:
validated that d35c59abb061a4a6fb18e82ac0862c26744d6ab5 is the commit of 5.5.0 in https://github.com/actions/setup-go/commits/v5.5.0